src/component/SortedLists.js

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import { FirestoreCollection, withFirestore } from 'react-firestore';
+import dayjs from 'dayjs';
+
+const SortedList = ({ token, handlePurchase, calculateIfPurchased }) => {
+  const concatPath = `/lists/${token}/items`;
+  const now = new Date();
+  const today = dayjs(now);
+
+  return (
+    <section className="listFrame">
+      <h1>&#128722;My Shopping List</h1>
+      {/* could also use &#128717; which is shopping bags */}
+      {/* // items that need to buy soon (fewer than seven days) */}
+      <FirestoreCollection
+        path={concatPath}
+        sort="numberOfDays:asc"
+        filter={['numberOfDays', '<', 7]}
+        render={({ isLoading, data }) => {
+          if (isLoading) {
+            return <div>Still Loading...</div>;
+          } else {
+            return (
+              <div id="soonItems">
+                <h2 className="itemsLabel">Soon Items</h2>
+                <ul>
+                  {data.map(item => (
+                    <li
+                      id={item.id}
+                      key={item.id}
+                      className={calculateIfPurchased(item)}
+                    >
+                      <div
+                        className={item.name}
+                        onClick={handlePurchase}
+                        id={item.id}
+                        aria-label="Soon Item"
+                        aria-required="true"
+                      >
+                        {item.name}
+                      </div>
+                    </li>
+                  ))}
+                </ul>
+              </div>
+            );
+          }
```

Comment:
No need to add the `else` statement, since you're already returning inside the `if` statement.

Reply:
I messed around with this syntax but kept getting errors. I'm not sure what you mean on this one.

Reply:
No problem, sorry I was not clear enough!

I’ll try to record a loom video later today before our meeting!

Reply:
@alejandronanez lost on this as well, do you mean the else after that is on line 22 on the above? I believe that's outdated now but just wanting to make sure I follow which else statement you're referring to.